### ISSUE TITLE
NullPointerException on Island Generation

### ISSUE THREAD

**Issue by kevinsal03** [rlf/uSkyBlock#1225]
_Please paste the output from `/usb version` below_
```
17:39:32.609
[CHAT] Name: uSkyBlock
17:39:32.609
[CHAT] Version: 2.8.3
17:39:32.609
[CHAT] Description: Ultimate SkyBlock v2.8.3-685d633
17:39:32.609
[CHAT] Language: en (en)
17:39:32.609
[CHAT]   State: d=128, r=64, i=6, p=5, n=true, awe=false
17:39:32.609
[CHAT] Server: Paper git-Paper-175 (MC: 1.14.4)
17:39:32.609
[CHAT]   State: online=true, bungee=true
17:39:32.609
[CHAT] ------------------------------
17:39:32.609
[CHAT] Vault 1.7.2-b107 (ENABLED)
17:39:32.610
[CHAT] WorldEdit 7.0.1-SNAPSHOT;4540-edb2e91 (ENABLED)
17:39:32.610
[CHAT] WorldGuard 7.0.0;02b731f (ENABLED)
17:39:32.692
[CHAT] Multiverse-Core 4.0.0-b766 (ENABLED)
17:39:32.693
[CHAT] Multiverse-NetherPortals 4.0.0-b757 (ENABLED)
17:39:32.693
[CHAT] ------------------------------
```

_What steps will reproduce the problem?_
1. Run server on PebbleHost
2. Attempt to generate an island via /is create
3. Plugin throws a NullPointerException during a task

_If you have any log-files, please paste them to [pastebin.com](http://pastebin.com)_
* server-log: https://pastebin.com/EeUPeFYp

Was not an issue when running on my local machine, downloaded the servers files from the server's FTP server and ran them locally with no changes and had no issues. Attempted to ask PebbleHost if they know anything of issues like and they seem to have no clue and just said work with the plugin dev.

**Comment by kevinsal03** [comment]
The issue seems to be resolvable by resetting the plugin completely and deleting the world.

**Comment by Muspah** [comment]
Thank you for taking the time to create an issue, I will check the code and see if we can prevent these NPE's in the future. Are islands being generated the way is should be for now on your end?

I've attached a copy of the logfile to this issue and removed the IP-addresses in it. You might want to do that yourself in the future if you don't want your IP-address floating around on the internet. There server is logging the IP that you're connecting from when you join the server.

[EeUPeFYp.txt](https://github.com/rlf/uSkyBlock/files/3537187/EeUPeFYp.txt)

**Comment by kevinsal03** [comment]
Yes by resetting the world it resolved the issue. Not the greatest solution, but since we are only in beta it was only a few players angered rather than many.

Thanks for editing the log to remove IP addresses. I completely blanked on that when making the issue, since I originally sent the log over to my hosting company I didn’t bother to remove them and totally forgot to do that when opening the issue here.

**Comment by Muspah** [comment]
No problem, I can't remove the log from Pastebin though, but you could remove the link from your message at least :)

**Comment by kevinsal03** [comment]
Already done, wish I had actually bothered to sign into Pastebin. The paste is unlisted and I’m going to see if I can request it be deleted.
